src/c++/perf_analyzer/genai-pa/genai_pa/test_end_to_end.py

```diff
@@ -0,0 +1,79 @@
+import subprocess
+import os
+import shutil
+import itertools
+
+import sys
+    
+
+# Define the options for option1 and option2
```

Comment:
Can you add comments explaining how to run this?

Reply:
Done. This is just a helper script and will be deleted once we have real E2E tests